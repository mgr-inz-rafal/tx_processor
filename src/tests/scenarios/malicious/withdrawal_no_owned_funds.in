type,client,tx,amount
deposit,1,1,1000
deposit,2,2,100
withdrawal,2,3,1000

