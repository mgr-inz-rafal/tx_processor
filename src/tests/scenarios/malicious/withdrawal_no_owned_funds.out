client,available,held,total,locked
1,1000,0,1000,false
2,100,0,100,false